doc/rbd/rbd-openstack.rst

```diff
@@ -232,13 +237,22 @@ Edit ``/etc/glance/glance-api.conf`` and add under the ``[glance_store]`` sectio
 
 For more information about the configuration options available in Glance please refer to the OpenStack Configuration Reference: http://docs.openstack.org/.
 
-Any OpenStack version
-~~~~~~~~~~~~~~~~~~~~~
-
 If you want to enable copy-on-write cloning of images, also add under the ``[DEFAULT]`` section::
 
     show_image_direct_url = True
 
+
+As of Mitaka
+~~~~~~~~~~~~
+
+To enable image locations and take advantage of copy-on-write cloning for images, add under the ``[DEFAULT]`` section::
+
+    show_multiple_locations = True
+
+
+Any OpenStack version
+~~~~~~~~~~~~~~~~~~~~~
+
 Note that this exposes the back end location via Glance's API, so the endpoint
```

Comment:
This section is related to the copy-on-write cloning advice. Perhaps break "Thin Provisioning" into its own main section with a "Mitaka" sub-section and a "Previous versions" sub-section?